### PR TITLE
Add doc to register a mgmt cluster in TMC

### LINKED ISSUE #3617: Add doc to Register a cluster with TMC
## Description

Add: 

    New topic for registering TMC
    Adds section for TMC step in the installer to Azure, Amazon, vSphere topics for deploying management cluster
    Adds a section to getting started guide for mgmt clusters on deregistering TMC

ref: https://github.com/vmware-tanzu/community-edition/pull/1650

### PR DESCRIPTION
**Pull request by kcoriordan**
## What this PR does / why we need it
Adds a new topic under Administration for Register a mgmt cluster in TMC
Adds a link to the new topic in the next steps topic at the end of the Getting Started guide.

## Which issue(s) this PR fixes

Fixes: #3617

### CODE REVIEW

File: docs/site/content/docs/tmc-register.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Register a Management Cluster in Tanzu Mission Control
+
```
Comment (joshrosso) [inline docs/site/content/docs/tmc-register.md:2]: Let's add a short description here of what TMC is.

Namely letting folks know that TMC is a VMware *product* (not free/open source project)

File: docs/site/content/docs/tmc-register.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    kubectl get po -A
+    ```
+
+    From the output copy the `vmware-system-tmc` namespace and run
+
+    ```sh
+    kubectl -n vmware-system-tmc get po 
+    ```
+
+1. Once all the pods associated with Tanzu Mission Control are running, in the Tanzu Mission Control console, you should be able to click View the Management Cluster, and click Verify Connection.
```
Comment (joshrosso) [inline docs/site/content/docs/tmc-register.md:53]: At the end here, let's add a "next steps" or something similar that points to the official TMC docs and tells users how to do new things.

File: docs/site/content/docs/tmc-register.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# Register a Management Cluster in Tanzu Mission Control
+
+VMware Tanzu Mission Control is a licensed VMWare product that you can integrate with.
+
+Use Tanzu Mission Control to manage your entire Kubernetes footprint, regardless of where your clusters reside.
+Bring all of your Kubernetes clusters together to monitor and manage them from a single console backed by an API-driven service.
+
+Complete the following steps to register a Tanzu Community Edition management cluster in Tanzu Mission Control.
```
Comment (kcoriordan) [inline docs/site/content/docs/tmc-register.md:8]: @joshrosso - Can you register mgmt clusters created in all the providers - including Docker?

File: docs/site/content/docs/tmc-register.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+1. Log into Tanzu Mission Control, for more information, see [Log In to the Tanzu Mission Control Console](https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-855A8998-E19A-46AC-A833-12C347486EF7.html) in the Tanzu Mission Control documentation.
```
Comment (joshrosso) [inline docs/site/content/docs/tmc-register.md:10]: for consistency, add a break here.

File: docs/site/content/docs/tmc-register.md (hunk 3 above)
Comment (joshrosso) [reply docs/site/content/docs/tmc-register.md:8]: @kcoriordan also worth mention, for vSphere clusters, TMC can only support clusters that use the *Photon* OS image. TMC will not work properly with vSphere clusters using the Ubuntu image.

Comment (kcoriordan) [reply docs/site/content/docs/tmc-register.md:16]: added this now

File: docs/site/content/docs/tmc-register.md (hunk 4 above)
Comment (kcoriordan) [reply docs/site/content/docs/tmc-register.md:18]: done

**Comment by kcoriordan** [comment]
@joshrosso - thanks for review, made those updates now

File: docs/site/content/docs/tmc-register.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    apply -f tmc-register.yaml
+    ```
+
+1. (Optional)  To watch the Tanzu Mission Control pods get created, run:
+
+    ```sh
+    kubectl get po -A
+    ```
+
+    From the output copy the `vmware-system-tmc` namespace and run
```
Comment (stmcginnis) [inline docs/site/content/docs/tmc-register.md:60]: I think this was from @joshrosso - what is the reason for needing to get all pods first before getting pods from the `vmware-system-tmc` namespace?

**Review by stmcginnis — APPROVED** [review]
Looks good to me. One question on an optional step, but nothing that I think should hold things up.

**Pull request merged**